scripts/healthcheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+paths=(
+  /usr/local/bin/agent 
+  /usr/local/bin/flowctl-go 
+  /usr/bin/gsutil 
+  /usr/local/bin/sops 
+  /usr/bin/jq 
+  /usr/local/bin/fetch-open-graph)
+
+for i in "${paths[@]}"
+do
```

Comment:
Seems like we should perhaps be checking to see if there's a running process for the `agent`?

Reply:
The reason I don't check the agent process is that it's pid 1. If pid 1 isn't running, the container isn't running.